bin/jsdoc-to-ts.js

```diff
@@ -4,10 +4,10 @@
 
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-const minimist = require('minimist');
-const jsdocToTs = require('..');
+import fs from 'fs';
+import path from 'path';
+import minimist from 'minimist';
+import jsdocToTs from '../index.js';
```

Comment:
Is this necessary to put `.js` at the end?

Reply:
Yes, it's one of the differences between ES modules and CommonJS.

https://nodejs.org/docs/latest-v12.x/api/esm.html#esm_mandatory_file_extensions
> A file extension must be provided when using the import keyword. Directory indexes (e.g. './startup/index.js') must also be fully specified.